src/main/res/values/strings.xml

```diff
@@ -191,18 +189,18 @@ limitations under the License.
     <string name="description_sensor_heart_rate">Heart rate (bpm)</string>
     <string name="description_sensor_power">Power (W)</string>
     <string name="description_default_speed_or_pace">By activity type</string>
-    <string name="description_speed_imperial">Speed (mi/hr)</string>
-    <string name="description_speed_metric">Speed (km/hr)</string>
+    <string name="description_speed_imperial">Speed (mi/h)</string>
+    <string name="description_speed_metric">Speed (km/h)</string>
     <string name="description_time">Time</string>
     <string name="description_total_distance">Total distance: %1$.2f km (%2$.1f mi)</string>
     <string name="description_total_time">Total time: %1$s</string>
     <string name="description_speed_source_gps">GPS</string>
     <!-- Export -->
     <string name="export_with_photos">with photos</string>
     <string name="export_without_photos">without photos</string>
-    <string name="export_error_post_workout">Post-workout export failed, please check the export directory.</string>
-    <string name="export_progress_message">Exporting to %1$s&#8230;</string>
-    <string name="export_track_already_exists_msg">The track %1$s already exists at the destination directory.</string>
+    <string name="export_error_post_workout">Could not export the post-workout. Check the export folder.</string>
```

Comment:
I think the meaning changed now. It's not a workout after a workout (post-workout), it's an export after the workout (post-workout-export).

Reply:
OK. What is being exported, and is it important that it happens "post-workout"?

Reply:
Right after the workout is finished some users want to have the recorded track automatically exported. So yes, it is important that the export of the track happens post-workout.

Reply:
```suggestion
    <string name="export_error_post_workout">Could not export track following newly finished workout . Check the export folder.</string>
```